src/app/app.module.ts

```diff
@@ -98,6 +99,7 @@ export function createTranslateLoader(http: Http) {
     TestPersistenceProvider,
     { provide: HAMMER_GESTURE_CONFIG, useClass: IonicGestureConfig },
     OutcomeBehaviourMapProvider,
+    CandidateDetailsCheckProvider,
```

Comment:
We should avoid global provider imports unless they are used in every page can we import it into the modules that use it

Reply:
The problem with that is that the provider data is not shared then, I tried importing it to two modules but since it has no idea that the candidate details was viewed or not

Reply:
Have we looked at using the store instead which is global by default?

Reply:
Using the state to store the data , will remove this provider and the import ;)